Gopkg.toml

```diff
@@ -25,7 +25,7 @@ ignored = ["github.com/Sirupsen/logrus"]
 
 [[constraint]]
   name = "github.com/DATA-DOG/godog"
-  version = "=0.7.6"
+  branch = "master"
```

Comment:
We need to put a revision to make sure this will not make the change to lock file when someone run `ensure`.